<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="navigationBG" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>171</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#navigationBG {
background-color: #ccdde8;

}

</string>
    </property>
    <widget class="QLabel" name="systemLabel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>15</y>
       <width>71</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#systemLabel {
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: #034488;
}</string>
     </property>
     <property name="text">
      <string>Movie
Ticketing
System</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/icon.png</normaloff>../img/icon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="adminButton">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>400</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#adminButton{
background-color: TRANSPARENT;
border: TRANSPARENT;
border-radius: 15px;
}

#adminButton:hover{
background-color: rgb(236, 236, 236);
}</string>
     </property>
     <property name="text">
      <string>Admin</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/admin.png</normaloff>../img/admin.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="movieButton_5">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>540</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/logout.png</normaloff>../img/logout.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="ticketsButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>199</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#ticketsButton{
background-color: TRANSPARENT;
border: TRANSPARENT;
border-radius: 15px;
}

#ticketsButton:hover{
background-color: rgb(236, 236, 236);
}</string>
     </property>
     <property name="text">
      <string>Bookings</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/tickets.png</normaloff>../img/tickets.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="crewButton">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>450</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#crewButton{
background-color: TRANSPARENT;
border: TRANSPARENT;
border-radius: 15px;
}

#crewButton:hover{
background-color: rgb(236, 236, 236);
}</string>
     </property>
     <property name="text">
      <string>Crew</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/crew.png</normaloff>../img/crew.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="cinemaButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#cinemaButton{
background-color: TRANSPARENT;
border: TRANSPARENT;
border-radius: 15px;
}

#cinemaButton:hover{
background-color: rgb(236, 236, 236);
}</string>
     </property>
     <property name="text">
      <string>Cinema</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/cinema.png</normaloff>../img/cinema.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="showButton">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>120</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#showButton{
background-color: TRANSPARENT;
border: TRANSPARENT;
border-radius: 15px;
}

#showButton:hover{
background-color: rgb(236, 236, 236);
}</string>
     </property>
     <property name="text">
      <string>Shows</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/movies.png</normaloff>../img/movies.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="movieButton">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>160</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#movieButton{
background-color: TRANSPARENT;
border: TRANSPARENT;
border-radius: 15px;
}

#movieButton:hover{
background-color: rgb(236, 236, 236);
}</string>
     </property>
     <property name="text">
      <string>Movies</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/movies.png</normaloff>../img/movies.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="genreButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#genreButton{
background-color: TRANSPARENT;
border: TRANSPARENT;
border-radius: 15px;
}

#genreButton:hover{
background-color: rgb(236, 236, 236);
}</string>
     </property>
     <property name="text">
      <string>Genres</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/genre.png</normaloff>../img/genre.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="mainBG" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>651</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#mainBG {
background-color: #fffcf6;

}

</string>
    </property>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>611</width>
       <height>491</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>609</width>
        <height>489</height>
       </rect>
      </property>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>611</width>
         <height>491</height>
        </rect>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::SelectedClicked</set>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="rowCount">
        <number>16</number>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>113</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>23</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>30</number>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column>
        <property name="text">
         <string>ADMIN_ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>FIRST NAME</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>MIDDLE NAME</string>
        </property>
        <property name="textAlignment">
         <set>AlignLeading|AlignVCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>LAST NAME</string>
        </property>
        <property name="textAlignment">
         <set>AlignLeading|AlignVCenter</set>
        </property>
       </column>
       <item row="0" column="1">
        <property name="text">
         <string/>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string/>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
       <item row="0" column="3">
        <property name="text">
         <string/>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>261</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../img/adminHeader.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="addTicket">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>60</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/addticket.png</normaloff>../img/addticket.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="editTicket">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>60</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/editTicket.png</normaloff>../img/editTicket.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteTicket">
     <property name="geometry">
      <rect>
       <x>537</x>
       <y>60</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/deleteTicket.png</normaloff>../img/deleteTicket.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <zorder>mainBG</zorder>
   <zorder>navigationBG</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
